src/main/java/model/LottoStatistics.java

```diff
@@ -0,0 +1,34 @@
+package model;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+
+public class LottoStatistics {
+    private static final int YIELD_SCALE = 3;
+
+    private Map<Rank, Integer> statistics;
+
+    public LottoStatistics(Map<Rank, Integer> statistics) {
+        this.statistics = statistics;
+    }
+
+    public double calculateYield(Money money) {
+        BigDecimal totalSpendMoney = money.calculateTotalSpendMoney();
+        BigDecimal totalBenefit = calculateTotalBenefit(statistics);
+        return totalBenefit.divide(totalSpendMoney, YIELD_SCALE, RoundingMode.HALF_EVEN).doubleValue();
+    }
+
+    private BigDecimal calculateTotalBenefit(Map<Rank, Integer> statistics) {
+        return Arrays.stream(Rank.values())
+                .map(rank -> rank.calculateBenefit(statistics.get(rank)))
+                .reduce(BigDecimal.ZERO, (previousBenefit, nextBenefit) -> previousBenefit.add(nextBenefit));
+    }
```

Comment:
이 또한 Rank 의 winningMoney 가 ``Money`` 타입이기 때문에 아래와 같이 변경해볼 수도 있을 것 같네요 😉

```suggestion
    private BigDecimal calculateTotalBenefit(Map<Rank, Integer> statistics) {
        return Arrays.stream(Rank.values())
                .map(rank -> rank.calculateBenefit(statistics.get(rank)))
                .reduce(BigDecimal.ZERO, (previousBenefit, nextBenefit) -> previousBenefit.add(nextBenefit));
    }
```